### DISCUSSION TITLE
Relab LX480 does not work on AG server-side windows 10

### DISCUSSION THREAD

**Discussion by JamesXXV** [apohl79/audiogridder#930] [Help]
<img width="263" alt="Screen Shot 2022-03-23 at 9 17 10 PM" src="https://user-images.githubusercontent.com/94138763/159742281-9e3e32fd-4b65-4f4a-9650-18a82b4c26de.png">

As shown in the above picture, I cannot get this CPU heavy reverb to load, is there something I can do to fix it?f

**Comment by oldtos** [comment]
I had a similar issue.....this plugin has some known issues.. some of the issues were fixed by "others" ... I did get it to load and work fine when I loaded blue cat mb-7 in AudioGridder  then loaded LX480 in MB-7.....blue cat patch work may work as well..
